tests/e2e/default/wdio.conf.js

```diff
@@ -168,7 +172,7 @@ const baseConfig = {
   // See the full list at http://mochajs.org/
   mochaOpts: {
     ui: 'bdd',
-    timeout: 120000,
+    timeout: debug ? (24 * 60 * 60 * 1000) : DEFAULT_TIMEOUT,
```

Comment:
Can you please declare `24 * 60 * 60 * 1000` separately as DEBUG_TIMEOUT? and add a comment about what this value represents.